docs/Deployment.md

```diff
@@ -81,26 +81,26 @@ Once the `IMMUTABLE_CREATE2_FACTORY_ADDRESS` exists, begin to deploy the contrac
 ```
 
 3. Validate deployments were successful by checking that `Seaport` is returned:
 
 ```
-cast call --rpc-url ${RPC_URL} 0x00000000000000ADc04C56Bf30aC9d3c0aAF14dC 'name()(string)'
+cast call --rpc-url ${RPC_URL} 0x0000000000000068F116a894984e2DB1123eB395 'name()(string)'
 ```
 
 ## Verifying Seaport and ConduitController
 After `Seaport` and `ConduitController` are deployed, they are verified as follows:
 
 1. Ensure that `EXPLORER_API_KEY` and `NETWORK_RPC` are set in `.env` appropriately.
-2. Navigate to the `1.1` release tag on the Seaport repo and build artifacts:
+2. Navigate to the `1.6` release tag on the Seaport repo and build artifacts:
```

Comment:
this should ideally be performed from the 1.6 release on the `seaport-core` repo